app/src/org/commcare/android/util/FileUtil.java

```diff
@@ -395,4 +396,20 @@ public static String getDumpDirectory(Context c){
                 return null;
             }
         }
+
+    public static Properties loadProperties(Context c) throws IOException {

```

Comment:
Don't we already have a lot of property-like loading, such as android preferences? We can always generalize property loading in the future, but since we aren't sure that we will ever need it, I think it is better to be very specific.

I would recommend changing the method name to 'loadAcraProperties', moving it to ACRAUtils, and changing the name of the properties file to  'acra.properties'.

Feel free to disagree and leave this here, but that is my position on this.